setup.py

```diff
@@ -76,7 +76,9 @@ def read(path):
     extras_require=dict(
         test=[
             'crate[test]',
-            'zc.customdoctests<2'
+            'zc.customdoctests<2',
+            # FIXME once tested and merged, fix the name
```

Comment:
NB! Needs to be updated before the merge

Reply:
Thank you. See also https://github.com/crate/crash/pull/410#issuecomment-1824711962:

> 3. @pilosus: When following the (in my opinion) right strategy outlined at 2., I am humbly asking for your patience on this. It would need to wait a bit for the testcontainers code to be upstreamed beforehand, [...]

\[ on which you have worked on behalf of https://github.com/crate-workbench/cratedb-toolkit/pull/82 \].

Reply:
I've adjusted the dependencies using 3191babfa6a.

Reply:
We observe that the test adapter may need a few more improvements, so we postponed the plan about the upstreaming. We released the most recent updates with [cratedb-toolkit](https://pypi.org/project/cratedb-toolkit/) 0.0.4. Can I ask you to upgrade?

Reply:
Done